src/styles/components/sidebar.scss

```diff
@@ -195,3 +196,97 @@ $sidebar-item-font-size: 14px;
 		}
 	}
 }
+
+.modal {
+	.modal-dialog.left, {
+		position: fixed;
+		margin: auto;
+		width: 200px;
+		height: 100%;
+		-webkit-transform: translate3d(0%, 0, 0);
+		-ms-transform: translate3d(0%, 0, 0);
+		-o-transform: translate3d(0%, 0, 0);
+		transform: translate3d(0%, 0, 0);
+	}
+
+	.modal-dialog.left {
+		.modal-content {
+			height: 100%;
+		}
+	}
+
+	.modal-content {
+		display: block;
+		overflow: hidden;
+	}
+
+	.modal-dialog.left {
+		.modal-body {
+			padding: 15px 15px 80px;
+		}
+	}
+
+	/*Left*/
+	&.fade {
+		.modal-dialog.left {
+			left: -200px;
+			-webkit-transition: opacity 0.3s linear, left 0.3s ease-out;
+			-moz-transition: opacity 0.3s linear, left 0.3s ease-out;
+			-o-transition: opacity 0.3s linear, left 0.3s ease-out;
+			transition: opacity 0.3s linear, left 0.3s ease-out;
+		}
+	}
+	&.fade.show {
+		.modal-dialog.left {
+			left: 0;
+		}
+	}
+}
+
+@media screen and (max-width: 800px) {
+	.mobile-sidebar-burger {
+		color: $toggle-text-color;
+		visibility: visible;
+		font-size: 1.5rem;
```

Comment:
@Aringoaway Where did you take this icon size? As far as I know in Header, we have icons `1.1rem`

Reply:
I think it should be the same everywhere (thus also in this toggler arrow for header dropdown)

Reply:
If possible, please change it there as well

Reply:
@Pe5h4  It's a size of toggle button, 1.5 rem
![image](https://user-images.githubusercontent.com/65866093/181167404-58d47bbc-742a-4c63-83eb-5a72615cce82.png)

Reply:
@Aringoaway I know, that's why I wrote you to change it as well. We need to have some consistency in item sizes, right? But lets discuss that on standup